meson.build

```
@@ -0,0 +1,714 @@
+project('geany', 'c', 'cpp', version: '1.38')
```

```suggestion
project('geany', 'c', 'cpp', version: '1.38', default_options: 'warning_level=2')
```
https://mesonbuild.com/Builtin-options.html#core-options